client/my-sites/people/edit-team-member-form/index.jsx

```diff
@@ -1,42 +1,44 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	LinkedStateMixin = require( 'react-addons-linked-state-mixin' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' ),
-	debug = require( 'debug' )( 'calypso:my-sites:people:edit-team-member-form' ),
-	omit = require( 'lodash/omit' ),
-	assign = require( 'lodash/assign' ),
-	filter = require( 'lodash/filter' ),
-	pick = require( 'lodash/pick' ),
-	page = require( 'page' );
+import React from 'react';
+import LinkedStateMixin from 'react-addons-linked-state-mixin';
+import PureRenderMixin from 'react-pure-render/mixin';
+import debugModule from 'debug';
+import omit from 'lodash/omit';
+import assign from 'lodash/assign';
+import filter from 'lodash/filter';
+import pick from 'lodash/pick';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-var Main = require( 'components/main' ),
-	HeaderCake = require( 'components/header-cake' ),
-	Card = require( 'components/card' ),
-	FormLabel = require( 'components/forms/form-label' ),
-	FormFieldset = require( 'components/forms/form-fieldset' ),
-	FormTextInput = require( 'components/forms/form-text-input' ),
-	FormButton = require( 'components/forms/form-button' ),
-	FormButtonsBar = require( 'components/forms/form-buttons-bar' ),
-	PeopleProfile = require( 'my-sites/people/people-profile' ),
-	UsersStore = require( 'lib/users/store' ),
-	UsersActions = require( 'lib/users/actions' ),
-	user = require( 'lib/user' )(),
-	protectForm = require( 'lib/mixins/protect-form' ),
-	DeleteUser = require( 'my-sites/people/delete-user' ),
-	PeopleNotices = require( 'my-sites/people/people-notices' ),
-	PeopleLog = require( 'lib/people/log-store' ),
-	analytics = require( 'lib/analytics' ),
-	RoleSelect = require( 'my-sites/people/role-select' );
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import FormLabel from 'components/forms/form-label';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import PeopleProfile from 'my-sites/people/people-profile';
+import UsersStore from 'lib/users/store';
+import UsersActions from 'lib/users/actions';
+import user from 'lib/user';
```

Comment:
we removed the `()` call to `require( 'lib/user' )`, I am getting a ` _user2.default.get is not a function` error when testing :)